src/posix.mak

```diff
@@ -2,46 +2,74 @@
 include osmodel.mak
 
 ifeq (,$(TARGET_CPU))
-    $(info no cpu specified, assuming X86)
-    TARGET_CPU=X86
+    $(info No cpu specified, assuming X86)
+    TARGET_CPU := X86
 endif
 
 ifeq (X86,$(TARGET_CPU))
-    TARGET_CH = $C/code_x86.h
-    TARGET_OBJS = cg87.o cgxmm.o cgsched.o cod1.o cod2.o cod3.o cod4.o ptrntab.o
+    TARGET_OBJS := cg87.o cgxmm.o cgsched.o cod1.o cod2.o cod3.o cod4.o ptrntab.o
 else
     ifeq (stub,$(TARGET_CPU))
-        TARGET_CH = $C/code_stub.h
-        TARGET_OBJS = platform_stub.o
+        TARGET_OBJS := platform_stub.o
     else
-        $(error unknown TARGET_CPU: '$(TARGET_CPU)')
+        $(error Unknown TARGET_CPU: '$(TARGET_CPU)')
     endif
 endif
 
-INSTALL_DIR=../../install
-# can be set to override the default /etc/
-SYSCONFDIR=/etc/
-
-C=backend
-TK=tk
-ROOT=root
-
 ifeq (osx,$(OS))
     export MACOSX_DEPLOYMENT_TARGET=10.3
 endif
-LDFLAGS=-lm -lstdc++ -lpthread
 
-#ifeq (osx,$(OS))
-#	HOST_CC=clang++
-#else
-	HOST_CC=g++
-#endif
-CC=$(HOST_CC) $(MODEL_FLAG)
-GIT=git
 
-# Compiler Warnings
+# Directory configuration
+########################################################################
+INSTALL_DIR := ../../install
```

Comment:
I typically default to using := since it behaves more like ordinary assignment. It has worked well through the years. The delayed assignment often confuses people in my opinion. But yes there are occations when its is necessary to use only =. A good way to most often get what one expect is to use := when no variables are occuring on the RHS, and = when they are.

This posix.mak is not overly complex so missbehaving delayed assignment should be a less of a problem. I will refrain from using it more :). Any particular reverts?